pandas/core/tools/datetimes.py

```diff
@@ -1241,60 +1198,6 @@ def coerce(values):
     return values
 
 
-def _attempt_YYYYMMDD(arg: npt.NDArray[np.object_], errors: str) -> np.ndarray | None:
```

Comment:
i think there's an issue about this fastpath not actually being fast. if this is removed, the issue is probably closeable

Reply:
Unfortunately, not quite

I've left a comment there, but removing it wouldn't fix the issue because 6-digit %Y%m%d dates wouldn't get parsed